<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuyOrderWidget</class>
 <widget class="QWidget" name="BuyOrderWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>510</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>120</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 19px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>买入</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>670</width>
     <height>280</height>
    </rect>
   </property>
   <widget class="QLabel" name="priceLabel2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>价格</string>
    </property>
   </widget>
   <widget class="QLabel" name="priceLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="amountLineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>143</y>
      <width>280</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="cancelBtn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>220</y>
      <width>96</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QToolButton" name="allBtn">
    <property name="geometry">
     <rect>
      <x>263</x>
      <y>145</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color: rgb(83,61,138);
font: 11px &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>全部兑换</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>支付数量</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>手续费</string>
    </property>
   </widget>
   <widget class="QToolButton" name="okBtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>96</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>合约地址</string>
    </property>
   </widget>
   <widget class="QLabel" name="contractAddressLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="feeLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>47</y>
      <width>201</width>
      <height>145</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>343</x>
      <y>47</y>
      <width>20</width>
      <height>151</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="tipLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>合约执行错误</string>
    </property>
   </widget>
  </widget>
  <zorder>widget</zorder>
  <zorder>label_5</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
